src/localization/bundles/en.json

```diff
@@ -71,7 +71,7 @@
     "id.with_colon": "ID:",
     "common.encryptkey.message": "Please provide encryption key for running playbooks",
     "common.encryptkey.title": "Provide encryption key",
-    "common.encryptkey.description": "The operation you are about to perform requires the encryption key. Please provide the encryption key.",
+    "common.encryptkey.description": "The operation you are about to perform requires the encryption key. Please provide the encryption key. If provide wrong encryption key, operation will fail. Refresh the page to invoke this dialog again in case wrong key was provided. ",
```

Comment:
The wording here needs a little fine-tuning. Consider:

The operation you are about to perform runs a playbook that requires the encryption key. If you provide wrong encryption key, the playbook will fail and you will need to refresh your browser window in order to re-enter the encryption key.

Reply:
ok, will update. Thanks for the wordings fixes.